java-api-core/src/com/kinvey/java/AbstractClient.java

```diff
@@ -77,6 +77,11 @@
     
     private GenericData customRequestProperties = new GenericData();
 
+    /**
+     * Value that represents state if delta set caching should be enabled
+     */
+    private boolean useDeltaCache;
```

Comment:
@misha-vakulich Is there a default value set for `useDeltaCache`?  Please make sure it is set to `false` by default.